scripts/benchmark_multithreaded.py

```diff
@@ -0,0 +1,205 @@
+# To prevent _tkinter.TclError: https://stackoverflow.com/a/37605654
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+import os
+import argparse
+import multiprocessing
+import subprocess
+import json
+
+from datetime import datetime
+
```

Comment:
```
Markus.Dreseler@nemea:~/hyrise/build-release$ python3 ../scripts/benchmark_multithreaded.py --runs 1000 -q 1 2 ./hyriseBenchmarkTPCH
usage: benchmark_multithreaded.py [-h] [-v] [-q Q [Q ...]] [--result-dir DIR]
                                  [--result-name NAME] [-c C [C ...] |
                                  --max-cores C] (-r N | --runs-per-core N)
                                  EXECUTABLE
benchmark_multithreaded.py: error: argument -q/--queries: invalid int value: './hyriseBenchmarkTPCH'
```

Can we do anything about this?

Reply:
We could make the executable a named argument `-e, --executable` instead of a positional argument?

Reply:
hmpf. That would mean that if we want to pass parameters to the executable, we would have to enclose it in quotes, right?

And what if, instead of using Python conventions, we use the same query parameter format that Hyrise uses?

Reply:
hmpf. That would mean that if we want to pass parameters to the executable, we would have to enclose it in quotes, right?

And what if, instead of using Python conventions, we use the same query parameter format that Hyrise uses?

Reply:
Sounds good.